examples/build.gradle

```diff
@@ -30,6 +30,16 @@ dependencies {
     sparklingWaterAssemblyJar project(path: ':sparkling-water-assembly', configuration: 'shadow')
     compile project(path: ':sparkling-water-assembly', configuration: 'shadow')
 
+    // H2O HDFS Persistent layer
```

Comment:
For Some reason, Spark 2.1 requires this dependency in order to find some apache classes, will investigate later why